release_creation/snapshot/download.sh

```diff
@@ -0,0 +1,6 @@
+python -m pip install --upgrade pip
+rm -rf $1
+mkdir $1
+python -m pip download \
+ --dest $1 \
```

Comment:
Can we make these vars at top of script for sake of documenting what these params are intended to be from the calling script. `readonly` is fun for this.

Reply:
good call out!